wiki/Main_Page/de.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: main_page
+---
+
+<!-- Do not add any empty lines inside this div. -->
+
+<div class="wiki-main-page__blurb">
+Willkommen zu der osu! wiki. Ein Projekt was eine große Reichweite an osu! basierten Informationen beinhaltet.
+</div>
+
+<div class="wiki-main-page__panels">
+<div class="wiki-main-page-panel wiki-main-page-panel--full">
+
+# Erste Schritte
+
+[Willkommen](/wiki/Welcome) • [Installation](/wiki/Installation) • [Registrierung](/wiki/Registration) • [Hilfe-Center](/wiki/Help_Centre) • [FAQ](/wiki/FAQ)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Spiel-Client
+
+[Benutzeroberfläche](/wiki/Interface) • [Optionen](/wiki/Options) • [Visuelle Einstellungen](/wiki/Visual_Settings) • [Tastenkürzel-Referenzen](/wiki/Shortcut_key_reference) • [Konfigurationsdatei](/wiki/osu!_Program_Files/User_Configuration_File) • [Programmdateien](/wiki/osu!_Program_Files)
+
+[Dateiformate](/wiki/osu!_File_Formats): [.osz](/wiki/osu!_File_Formats/Osz_(file_format)) • [.osk](/wiki/osu!_File_Formats/Osk_(file_format)) • [.osr](/wiki/osu!_File_Formats/Osr_(file_format)) • [.osu](/wiki/osu!_File_Formats/Osu_(file_format)) • [.osb](/wiki/osu!_File_Formats/Osb_(file_format)) • [.db](/wiki/osu!_File_Formats/Db_(file_format))
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Spielverlauf
+
+[Spielmodis](/wiki/Game_mode): [osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania)
+
+[Beatmap](/wiki/Beatmap) • [Hit-Objekte](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Punktzahl](/wiki/Score) • [Wiederholung](/wiki/Replay) • [Multi](/wiki/Multi)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# [Beatmap editor](/wiki/Beatmap_Editor)
+
+Sections: [Zusammenstellung](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song-Einstellung](/wiki/Beatmap_Editor/Song_Setup)
+
+Komponenten: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Schlagschnellteiler](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Schlagschnellteiler](/wiki/Beatmap_Editor/Distance_Snap) • [Menu](/wiki/Beatmap_Editor/Menu) • [SB Ladung](/wiki/Beatmap_Editor/SB_Load) • [Zeitleisten](/wiki/Beatmap_Editor/Timelines)
```

Comment:
"Beat Snap Divisor" and "Distance snap" are two completely different terms, so they shouldn't be translated as the same word. Fwiw, the related translated articles don't translate these terms at all.

Reply:
_Beat Snap Divisor_ is translated as _Taktteiler_ in the game, which is accurate. And although I like translating words literally for fun, I think it's often more meaningful to use the English terms.
```suggestion
Komponenten: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Taktteiler](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance Snap](/wiki/Beatmap_Editor/Distance_Snap) • [Menü](/wiki/Beatmap_Editor/Menu) • [SB-Belastung](/wiki/Beatmap_Editor/SB_Load) • [Zeitleisten](/wiki/Beatmap_Editor/Timelines)
```